src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1513,6 +1513,24 @@ private String getPortFromMessageTokens(String[] messageTokens) throws PluginExe
         return null;
     }
 
+    private String findLocalPort(String internalContainerPort) {
+        String dockerPortCmd = "docker port " + containerName + " " + internalContainerPort;
+        String cmdResult = execDockerCmd(dockerPortCmd, 10, false);
+        if (cmdResult == null) {
+            error("Unable to retrieve locally mapped port.");
+            return null;
+        }
+        String[] cmdResultSplit = cmdResult.split("RC=");
+        if (cmdResultSplit[cmdResultSplit.length - 1].equals("1")) {
```

Comment:
It might be better to check if this is not 0, since conventionally any non-zero return code could also indicate error.